### PR TITLE
Allow passing desired name while creating account.

### LINKED ISSUE #198: Account import should allow the user to enter account name
Not everybody wants to name their first imported account Alice...

### PR DESCRIPTION
**Pull request by muzam1l**
Fixes #198

### CODE REVIEW

**Review by valstu — CHANGES_REQUESTED** [review]
What do you think, would it make sense to add name field to import account as well? 🤔

**Review by valstu — CHANGES_REQUESTED** [review]
What do you think, would it make sense to add name field to import account as well? 🤔

**Comment by vbar** [comment]
> What do you think, would it make sense to add name field to import account as well? thinking

Yes, I think it makes sense especially for import...

File: components/Accounts.tsx [hunk 1]
```diff
@@ -514,31 +513,67 @@ const Accounts: FC<AccountProps> = (props) => {
...
+
+  const btnText = type === "import" ? "Import" : "Create";
+  const title = type === "import" ? "Import Account" : "Create Account";
+  const labelText = type === "import" ? "Account secret" : "Account Name";
+  const cancelText = type === "import" ? "Cancel" : "Skip";
+
+  const handleSubmit = () => {
+    if (type === "create") {
+      const name = capitalize(value);
+      addFaucetAccount(name, true);
```
[the remark this answers is not in the document]
Comment (muzam1l) [reply components/Accounts.tsx:536]: Yes!

**Review by vbar — CHANGES_REQUESTED** [review]
It's very nonobvious to me that I don't _have_ to fill in the name - I think the dialog should include a comment (or help popup, or something) to that effect...

**Comment by muzam1l** [comment]
Okay to merge?

**Comment by vbar** [comment]
> Okay to merge?

Ah, that's why it isn't on hooks-builder.xrpl.org - it hadn't been merged yet... I'd say yes, but on Monday... :-)

**Pull request merged**